api/tests/Data/Utils/MockContainerProvider.cs

```diff
@@ -60,6 +60,8 @@ public static Container CreateMockContainer(List<T> data)
                     });
 
             container.GetItemLinqQueryable<T>().Returns(data.AsQueryable());
+            
+            container.GetItemLinqQueryable<T>(Arg.Any<bool>()).Returns(data.AsQueryable());
```

Comment:
Why is this needed?

Reply:
The clients are calling GetItemLinqQueryable(true), which is returning null since that method isn't mocked

Reply:
I believe some tests fail on main atm